runtimevar/blobvar/blobvar.go

```diff
@@ -34,6 +51,90 @@ import (
 	"gocloud.dev/runtimevar/driver"
 )
 
+func init() {
+	runtimevar.DefaultURLMux().RegisterVariable(Scheme, &lazyBucketOpener{})
+}
+
+// Scheme is the URL scheme blobvar registers its URLOpener under on runtimevar.DefaultMux.
+const Scheme = "blob"
+
+type lazyBucketOpener struct {
```

Comment:
nit: There isn't a lazy evaluation here, really. I think that this global could probably use the same `URLOpener`.

Reply:
Done.